agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/statsbeat/Feature.java

```diff
@@ -49,7 +49,9 @@
   SPRING_INTEGRATION_DISABLED(19),
   LEGACY_PROPAGATION_DISABLED(20),
   GRIZZLY_DISABLED(21), // preview instrumentation
-  STATSBEAT_DISABLED(22); // disable non-essential statsbeat
+  STATSBEAT_DISABLED(22), // disable non-essential statsbeat
+  QUARTZ_DISABLED(23), // preview instrumentation
+  APACHE_CAMEL_DISABLED(24); // preview instrumentation
```

Comment:
APACHE_CAMEL_ENABLED?

Reply:
just following convention, do you want to change it for existing too?

Reply:
yes, please.

Reply:
or i can clean up after your pr